### ISSUE TITLE
Error downloading `codeium` server via `plenary.nvim` on Windows

### ISSUE THREAD

**Issue by CharlesChiuGit** [Exafunction/codeium.nvim#44]
Here's the error:
![圖片](https://user-images.githubusercontent.com/32497323/230577807-8c64e1c9-ea91-4990-8cf7-91e3428cd2df.png)

So I trace back to `plenary.nvim` and found a [relative issue](https://github.com/nvim-lua/plenary.nvim/issues/176)
It seems like `curl` with `--compressed` is not supported on `curl 8.0.1` (I install curl via `scoop`)
![圖片](https://user-images.githubusercontent.com/32497323/230581272-4b637f62-a819-4290-a978-a0036282058b.png)

I not sure if this error should be upstream to `plenary` or not.

**Comment by jcdickinson** [comment]
I can prevent it from using `--compressed`, and will look into that this weekend.

**Comment by CharlesChiuGit** [comment]
tks!

**Comment by jcdickinson** [comment]
Please let me know if this has been fixed with the latest commit.

**Comment by CharlesChiuGit** [comment]
Thanks for the quick fix! Codeium server could now be installed on Windows now. Thanks!
![圖片](https://user-images.githubusercontent.com/32497323/230818698-ea1c6d7b-9f32-48df-92b3-6e236a9d4bb5.png)

**Comment by CharlesChiuGit** [comment]
Hmmm, unfortunately it seems like sth else is still broken. Every time I re-open neovim, `codeium.nvim` keeps downloading the server as the above image.

Also when I tried to auth, it won't work. Here's the error log:
![圖片](https://user-images.githubusercontent.com/32497323/230821305-416288cf-03a6-4c03-ab7d-368b3c2aedf5.png)

```log
Error executing Lua callback: ...cal/nvim-data/site/lazy/plenary.nvim/lua/plenary/job.lua:106: uuidgen: Executable not found

stack traceback:
        ...cal/nvim-data/site/lazy/plenary.nvim/lua/plenary/job.lua:106: in function 'job'
        ...ocal/nvim-data/site/lazy/codeium.nvim/lua/codeium/io.lua:191: in function 'get_command_output'
        ...ocal/nvim-data/site/lazy/codeium.nvim/lua/codeium/io.lua:315: in function 'generate_uuid'
        ...cal/nvim-data/site/lazy/codeium.nvim/lua/codeium/api.lua:76: in function 'authenticate'
        ...al/nvim-data/site/lazy/codeium.nvim/lua/codeium/init.lua:20: in function <...al/nvim-data/site/lazy/codeium.nvim/lua/codeium/init.lua:17>
stack traceback:
        [C]: in function 'error'
        ...cal/nvim-data/site/lazy/plenary.nvim/lua/plenary/job.lua:106: in function 'job'
        ...ocal/nvim-data/site/lazy/codeium.nvim/lua/codeium/io.lua:191: in function 'get_command_output'
        ...ocal/nvim-data/site/lazy/codeium.nvim/lua/codeium/io.lua:315: in function 'generate_uuid'
        ...cal/nvim-data/site/lazy/codeium.nvim/lua/codeium/api.lua:76: in function 'authenticate'
        ...al/nvim-data/site/lazy/codeium.nvim/lua/codeium/init.lua:20: in function <...al/nvim-data/site/lazy/codeium.nvim/lua/codeium/init.lua:17>
```

**Comment by marvin89** [comment]
Hi,

I ran into the same issue and followed [these instructions](https://superuser.com/questions/1400372/what-part-of-the-windows-10-sdk-do-i-need-to-install-to-get-uuidgen). Make sure the `uuidgen` executable is in the path, I had to manually add in the environment variables.

I can now use `:Codeium Auth` which tells me that the API key was saved. I can also confirm I'm seeing Codeium server being downloaded every time the plugin loads.
